### ISSUE TITLE
Several consecutive days use the same word

### ISSUE THREAD

**Issue by JessFrancesB** [flackr/crosswordle#38]
I think someone went a bit mad with submissions and submitted a load of phrases with "French" in them. It's been nearly a week now, maybe longer. It's gotten to the point where "French" is the first word I guess. Is there any way to see how many of these are left, and to spread them out across the queue rather than have them on consecutive days? It's getting very tiring.

I don't actually know how the queue works so I don't know if there's a way to fix this without waiting it out or deleting the submissions.

**Comment by flackr** [comment]
Sorry to hear that it's getting tiring. It won't be too much longer - a few more days for which you'll know the first word and need to guess the second that matches it. It was a bit of an experiment and part of an intentional theme carried over several days, not an accident. I'm sure you've seen some of the other themes in the past (e.g. the 12 days of Christmas theme).

I have changed the queue in the past but this led to detrimental experiences due to some users having the previous list cached and resulting in different people solving different puzzles for the same day. While there are technical ways to fix this, this is a hobby project that I do in my spare time, so I'm generally avoiding changing the queue for anything less than a week out just to avoid this issue.

In short, I'm going to close this as it's only going to be a few more days, but I'll consider this experiment of using the same word across more than a few days as a negative experience in the future and avoid doing this again.

I didn't yet actually have a "submission" system, though you've got me thinking maybe I should! It can never be fully automated as I'd need to at least make sure the content is appropriate and sensible so I can always manually avoid duplication like this

**Comment by vinaykapadia** [comment]
I think that a themed sequence could be fun, such as Christmas. But to have the exact same word every single day got tiring.
